test/watcher.js

```diff
@@ -115,15 +115,20 @@ group('chokidar is installed', function (beforeEach, test, group) {
 
 		logger.clear.returns(true);
 
-		avaFiles = AvaFiles;
-
-		api.run.returns(new Promise(function () {}));
 		files = [
 			'test.js',
 			'test-*.js',
 			'test'
 		];
 
+		avaFiles = AvaFiles;
+
+		avaFiles.prototype.findTestFilesSync = function () {
+			return this.files;
```

Comment:
Because API expects resolved paths, `findTestFilesSync` needs to be stubbed to return "input" files without changes, in order to easily check them in tests. Otherwise, `findTestFilesSync` would return empty array.